### PR TITLE
New observables

### PR DESCRIPTION
**Pull request by sebdraven**

### CODE REVIEW

File: core/schemas/observables/file.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+class File(Observable):
+    value: str
+    type: ObservableType = ObservableType.file
+    name: str = None
+    size: int = None
+    sha256: SHA256 = None
+    md5: MD5 = None
+    sha1: SHA1 = None
+    mime_type: str = None
```
Comment (tomchop) [inline core/schemas/observables/file.py:16]: Be careful with the `None` default, as it makes all fields optional. If you really can't go any other way, at least specify why in the docstring. (I think this also deserves a docstring)

File: core/schemas/observables/registry_key.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from core.schemas.observable import Observable
+from core.schemas.observable import ObservableType
+
+class RegistryKey(Observable):
+    type: ObservableType = ObservableType.registry_key
+    key: str
+    value: str
+    hive: str | None = None
+    path_file: str | None = None
```
Comment (tomchop) [inline core/schemas/observables/registry_key.py:9]: What do you mean by this? 

I would also add a docstring to this class to explain what format things are expected to be in.

File: tests/apiv2/observables.py [hunk 3]
```diff
@@ -6,17 +6,15 @@
 
 from core.schemas.observable import Observable
 from core.web import webapp
+from core.schemas.observables.certificate import Certificate
```
Comment (tomchop) [inline tests/apiv2/observables.py:9]: Why the import here? You're not using it in the file

File: tests/observables_test.py
Comment (tomchop) [inline tests/observables_test.py:1]: This file is unused, let's port the changes to `tests/schemas/observable.py`

File: core/schemas/observables/file.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
+from core.schemas.observable import Observable
+from core.schemas.observable import ObservableType
+from core.schemas.observables.md5 import MD5
+from core.schemas.observables.sha1 import SHA1
+from core.schemas.observables.sha256 import SHA256
+
+
+class File(Observable):
+    value: str
```
[the remark this answers is not in the document]
Comment (sebdraven) [reply core/schemas/observables/file.py:9]: The last naming file:sha256

File: core/schemas/observables/certificate.py [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+
+class Certificate(Observable):
+    last_seen: datetime.datetime = Field(default_factory=datetime.datetime.now)
+    first_seen: datetime.datetime = Field(default_factory=datetime.datetime.now)
+    issuer: str | None = None
+    subject: str | None = None
+    serial_number: str | None = None
+    after: datetime.datetime | None = None
+    before: datetime.datetime | None = None
```
[the remark this answers is not in the document]
Comment (sebdraven) [reply core/schemas/observables/certificate.py:15]: Sometime in feed you have just à fingerprinting of certificate, so all fields can be to set at None

File: core/schemas/observables/file.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (tomchop) [reply core/schemas/observables/file.py:21]: Can you add this to the docstring?

**Pull request merged**